storm-core/src/jvm/backtype/storm/task/TopologyContext.java

```diff
@@ -154,6 +154,18 @@ public Fields getThisOutputFields(String streamId) {
     }
 
     /**
+     * Gets the declared output fields for the specified stream id for the component
+     * this task is a part of.
+     */
+    public Map<String, List<String>> getThisOutputFieldsForStreams() {
+    	Map<String, List<String>> streamToFields = new HashMap<String, List<String>>();
+    	for (String stream : this.getThisStreams()) {
+    		streamToFields.put(stream, this.getThisOutputFields(stream).toList());
```

Comment:
I'm not a big Java guy, so what's the appropriate spacing here?  Tabs or spaces? And how many?